src/@types/ProviderUser.ts

```diff
@@ -6,5 +6,6 @@ export type ProviderUser = {
   user: {
     id: string;
     identities: Identity[];
+    identity: Identity;
   };
 };
```

Comment:
Hmm.. I think this should be unchanged. The query should still return `Identity[]`, but with only one identity in it, based on the filter. Could you verify this?

Reply:
It returns an object
`{
  data: {
    provider: {
      id: 'xxxx-xxxx-xxxx-xxxx',
      name: 'Bluesports',
      user: [Object]
    }
  }
}`

Reply:
It's the `user` field that is useful to show :).

Reply:
It's still an object
`{
  id: 'xxxx-xxxx-xxxx-xxxx',
  identity: {
    id: 'xxxx-xxxx-xxxx-xxxx',
    primary: true,
    status: 'validated',
    type: 'email',
    value: 'xxx.xxx@xxx.xxx'
  }
}`

Reply:
You can go deeper in the object by calling it in your console.log.

Reply:
OK fixed 69a9c93